commons/src/main/java/org/dhis2/commons/dialogs/MediaDialog.kt

```diff
@@ -0,0 +1,341 @@
+private fun PreviewMediaDialog() {
+    MediaDialog(
+        title = "Tem local para lavar as mãos",
+        subTitle = "Pode ser uma torneira com água canalizada ou recepientes",
+        listOfAudioUrl = generateRandomURLs(),
+        listOfVideosUrl = generateRandomURLs(),
+        onMediaItemClicked = {},
+        onDismiss = {},
+    )
+}
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Preview(backgroundColor = 0xFFFFFFFF)
+@Composable
+private fun PreviewVideoMediaItem() {
+    MediaDialogItem(
+        type = MediaType.VIDEO,
+        title = "Como lavar as mãos",
+        url = generateRandomURLs()[0],
+        duration = generateRandomDuration(),
+        dateOfLastUpdate = generateRandomDate()
+    ) {}
+}
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Preview(backgroundColor = 0xFFFFFFFF)
+@Composable
+private fun PreviewAudioMediaItem() {
+    MediaDialogItem(
+        type = MediaType.AUDIO,
+        title = "Como lavar as mãos",
+        url = generateRandomURLs()[0],
+        duration = generateRandomDuration(),
+        dateOfLastUpdate = generateRandomDate()
+    ) {}
+}
+
+@RequiresApi(Build.VERSION_CODES.O)
+@Composable
+fun MediaDialog(
+    title: String,
+    subTitle: String,
+    listOfVideosUrl: List<String>, //Todo: create and use different object instead of String
+    listOfAudioUrl: List<String>, //Todo: create and use different object instead of String
+    onMediaItemClicked: (url: String) -> Unit,
+    onDismiss: () -> Unit,
+) {
+    Dialog(onDismissRequest = { onDismiss.invoke() }) {
+        Surface(
+            shape = RoundedCornerShape(16.dp),
+            color = Color.White
+        ) {
+            Box(contentAlignment = Alignment.Center) {
+                Column(modifier = Modifier.padding(20.dp)) {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        horizontalArrangement = Arrangement.Center,
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        MediaDialogMainIcon()
+                    }
+                    Spacer(modifier = Modifier.height(16.dp))
+
+                    Text(
+                        text = title,
+                        modifier = Modifier.fillMaxWidth(),
+                        textAlign = TextAlign.Center,
+                        fontSize = 20.sp,
+                    )
+                    Spacer(modifier = Modifier.height(16.dp))
+
+                    Text(
+                        text = subTitle,
+                        fontSize = 12.sp,
+                        textAlign = TextAlign.Center,
+                        modifier = Modifier.fillMaxWidth(),
+                        color = colorResource(id = R.color.colorPrimary_f57)
+                    )
+                    Spacer(modifier = Modifier.height(16.dp))
+
+                    LazyColumn(modifier = Modifier.fillMaxWidth()) {
+
+                        for (videoUrl in listOfVideosUrl) {
```

Comment:
You don't need to use for loop for listing data, you can replace it with:

LazyColumn {
    items(yourList) {
        yourComponent()
    }
}